proto/catalog.proto

```diff
@@ -32,6 +32,13 @@ enum SchemaRegistryNameStrategy {
   TOPIC_RECORD_NAME_STRATEGY = 2;
 }
 
+enum StreamJobStatus {
+  // Prefixed by `STREAM_JOB_STATUS` due to protobuf namespacing rules.
+  STREAM_JOB_STATUS_UNSPECIFIED = 0;
```

Comment:
```suggestion
  UNSPECIFIED = 0;
```

Reply:
Protobuf namespacing rules follow c++. `SinkType` already has `UNSPECIFIED` as a variant, so it will conflict.

Reply:
```
  Failed to compile grpc!: Custom { kind: Other, error: "protoc failed: catalog.proto:95:3: \"UNSPECIFIED\" is already defined in \"catalog\".\ncatalog.proto:95:3: Note that enum values use C++ scoping rules, meaning that enum values are siblings of their type, not children of it.  Therefore, \"UNSPECIFIED\" must be unique within \"catalog\", not just within \"SinkType\".\nbatch_plan.proto:5:1: Import \"catalog.proto\" was not found or had errors.\nbatch_plan.proto:66:3: \"catalog.StreamSourceInfo\" is not defined.\n" }
```

Reply:
Oh, I see. 👍 Just forget it and get this PR merged first.